src/components/DisplayInput.js

```diff
@@ -33,11 +35,16 @@ class DisplayInput extends React.Component {
     valid: true
   };
 
-  _isLargeOrMediumWindowSize = () => {
+  componentWillMount () {
+    this._labelId = _uniqueId('DI');
+    this._inputId = _uniqueId('DI');
```

Comment:
this is used to associate the label with the input, introduced here  https://github.com/mxenabled/mx-react-components/pull/559